pkg/dryrun/validation.go

```diff
@@ -227,6 +243,7 @@ func (d *DryRun) deepEqualEnvVars(env1, env2 []v1.EnvVar) error {
 			"NODE_NAME":            true,
 			"PX_NAMESPACE":         true,
 			"PX_SECRETS_NAMESPACE": true,
+			"controller_sn":        true,
```

Comment:
What is controller_sn? Why it's all lowercase?

Reply:
it's an env variable required by ccm container, daemonset spec gen just pass through